task/dev3/Operations/InformationPrinter.java

```diff
@@ -0,0 +1,67 @@
+package task.dev3.Operations;
+
+import task.dev3.User.*;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * Print options information.
+ */
+public class InformationPrinter {
```

Comment:
Badly designed class. All operation must have the same interface so you can go through it in one loop